### DISCUSSION TITLE
Maps for popular Packs?

### DISCUSSION THREAD

**Discussion by tobar-io** [Orange-Panda/Rewired-Glyphs#7] [General]
Hi. First of all, I really love the package. I used it in all my projects and it just works. But the pixel art style of kennys asset doesnt fit my games style and I wonder if there are any ready maps like for Xelus package that I can use, before I reassign every glyph myself, which I'm doing right now and it takes forever 😄 

If not, maybe I can provide something when I'm done. I'm doing https://mrbreakfastsdelight.itch.io/mr-breakfasts-free-prompts atm.

Kenneys HQ Pack would be an obvious contender I would really like to use: https://kenney.nl/assets/input-prompts

**Comment by Orange-Panda** [comment]
I agree the visual style of the current example's pixel art style is a bit limiting for non-pixel art projects. I'll add some more examples that use other input prompt packs which are a bit more universal in style

**Reply by Orionhart** [reply]
These are also quite good. I may end up having to make a map for this because I'm using it at work.
https://thoseawesomeguys.com/prompts/

**Comment by Orange-Panda** [comment]
A new release [2.0.0](https://github.com/Orange-Panda/Rewired-Glyphs/discussions/13) has been published which includes a new sample using the [Xelu](https://thoseawesomeguys.com/prompts/) prompts

Edit: Also an article has been written for the documentation website that details [the process of creating a Glyph Collection](https://orange-panda.github.io/Rewired-Glyphs/pages/manual/creating-collection.html) which should assist in the production of glyph packs

**Reply by tobar-io** [reply]
Fantastic, well done! Will try it on my next project for sure, but I'm too terrified to update my current one 🦞

**Reply by Orionhart** [reply]
Oh awesome! Not sure if you were already working on it before my comment posting the link but very glad to see.
